plugins/modules/remote_management/redfish/ilo_redfish_config.py

```diff
@@ -0,0 +1,175 @@
+      baseuri: 15.X.X.X
+      username: Admin
+      password: Testpass123
+      attribute_name: TimeZone
+      attribute_value: Chennai
+'''
+
+RETURN = '''
+msg:
+    description: Message with action result or error description
+    returned: always
+    type: str
+    sample: "Action was successful"
+'''
+
+CATEGORY_COMMANDS_ALL = {
+    "Manager": ["SetTimeZone", "SetDNSserver", "SetDomainName", "SetNTPServers", "SetWINSReg"]
+}
+
+from ansible_collections.community.general.plugins.module_utils.ilo_redfish_utils import iLORedfishUtils
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+
+
+def main():
+    result = {}
+    module = AnsibleModule(
+        argument_spec=dict(
+            category=dict(required=True, choices=list(
+                CATEGORY_COMMANDS_ALL.keys())),
+            command=dict(required=True, type='list', elements='str'),
+            baseuri=dict(required=True),
+            username=dict(),
+            password=dict(no_log=True),
+            auth_token=dict(no_log=True),
+            attribute_name=dict(required=True),
+            attribute_value=dict(),
+            timeout=dict(type='int', default=10)
+        ),
+        required_together=[
+            ('username', 'password'),
+        ],
+        required_one_of=[
+            ('username', 'auth_token'),
+        ],
+        mutually_exclusive=[
+            ('username', 'auth_token'),
+        ],
+        supports_check_mode=False
+    )
+
+    category = module.params['category']
+    command_list = module.params['command']
+
+    creds = {"user": module.params['username'],
+             "pswd": module.params['password'],
+             "token": module.params['auth_token']}
+
+    timeout = module.params['timeout']
+
+    root_uri = "https://" + module.params['baseuri']
+    rf_utils = iLORedfishUtils(creds, root_uri, timeout, module)
+    mgr_attributes = {'mgr_attr_name': module.params['attribute_name'],
+                      'mgr_attr_value': module.params['attribute_value']}
+
+    offending = [
+        cmd for cmd in command_list if cmd not in CATEGORY_COMMANDS_ALL[category]]
+
+    if offending:
+        module.fail_json(msg=to_native("Invalid Command(s): '%s'. Allowed Commands = %s" % (
+            offending, CATEGORY_COMMANDS_ALL[category])))
+
+    if category == "Manager":
+        resource = rf_utils._find_managers_resource()
+        if not resource['ret']:
+            module.fail_json(msg=to_native(resource['msg']))
+
+        dispatch = dict(
+            SetTimeZone=rf_utils.set_time_zone,
+            SetDNSserver=rf_utils.set_dns_server,
+            SetDomainName=rf_utils.set_domain_name,
+            SetNTPServers=rf_utils.set_ntp_server,
+            SetWINSReg=rf_utils.set_wins_registration
+        )
+
+        for command in command_list:
+            result[command] = dispatch[command](mgr_attributes)
+
+    module.exit_json(ilo_redfish_config=result)
```

Comment:
You need to return whether something changed. I guess that this should be `True` as soon as any command has been executed?
```suggestion
    module.exit_json(ilo_redfish_config=result, changed=changed)
```